### PR TITLE
Update offcanvas example to include .show class

### LINKED ISSUE #33863: V5 Offcanvas : Missing .show class in docs
there is a missing class .show in offcanvas documentation

https://getbootstrap.com/docs/5.0/components/offcanvas/

**current example :** 

```
<div class="offcanvas offcanvas-start" tabindex="-1" id="offcanvas" aria-labelledby="offcanvasLabel">
  <div class="offcanvas-header">
    <h5 class="offcanvas-title" id="offcanvasLabel">Offcanvas</h5>
    <button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close"></button>
  </div>
  <div class="offcanvas-body">
    Content for the offcanvas goes here. You can place just about any Bootstrap component or custom elements here.
  </div>
</div>
```

**updated example :**

```
<div class="offcanvas offcanvas-start show" tabindex="-1" id="offcanvas" aria-labelledby="offcanvasLabel" data-bs-backdrop="false" data-bs-scroll="true">
  <div class="offcanvas-header">
    <h5 class="offcanvas-title" id="offcanvasLabel">Offcanvas</h5>
    <button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close"></button>
  </div>
  <div class="offcanvas-body">
    Content for the offcanvas goes here. You can place just about any Bootstrap component or custom elements here.
  </div>
</div>
```

### PR DESCRIPTION
**Pull request by mdo**
Fixes #33863.

### CODE REVIEW

File: site/content/docs/5.1/components/offcanvas.md [hunk 1]
```diff
@@ -27,8 +27,20 @@ Offcanvas is a sidebar component that can be toggled via JavaScript to appear fr
 
 Below is an offcanvas example that is shown by default (via `.show` on `.offcanvas`). Offcanvas includes support for a header with a close button and an optional body class for some initial `padding`. We suggest that you include offcanvas headers with dismiss actions whenever possible, or provide an explicit dismiss action.
 
-{{< example class="bd-example-offcanvas p-0 bg-light overflow-hidden" >}}
-<div class="offcanvas offcanvas-start" tabindex="-1" id="offcanvas" aria-labelledby="offcanvasLabel">
```
Comment (GeoSot) [inline site/content/docs/5.1/components/offcanvas.md:31]: if you add `.show` class here, instead of rewrite the example snippet, is not fine?

Comment (mdo) [reply site/content/docs/5.1/components/offcanvas.md:31]: Adding `.show` to the example as-is would bring up the backdrop over the docs pages. Maybe we could do that and disable the backdrop and scrolling?

`data-bs-scroll="true" data-bs-backdrop="false"`

Comment (mdo) [reply site/content/docs/5.1/components/offcanvas.md:31]: Tried this, but the autofocus on the dismiss button scrolls the page then. 🤔  Thoughts?

Comment (julien-deramond) [reply site/content/docs/5.1/components/offcanvas.md:31]: You could try to add in _site/assets/js/application.js_ something like:
```js
// Offcanvas demo
 var myOffcanvas = document.querySelector('.bd-example-offcanvas #offcanvas');
 if (myOffcanvas) {
   myOffcanvas.addEventListener('show.bs.offcanvas', function(event){
     event.preventDefault();
   }, false);
 }
```
The advantage of keeping the `{{< example >}}` (if possible) is that we will continue to have the "Try it" button to go to StackBlitz.

**Pull request merged**